packages/transaction-controller/src/types.ts

```diff
@@ -689,3 +692,38 @@ export type DefaultGasEstimates = {
    */
   maxPriorityFeePerGas?: string;
 };
+
+/**
+ * Represents an error encountered during a transaction.
+ */
+export type TxError = {
```

Comment:
Should we call this `TransactionError` to be consistent with the other `TransactionX` types?